auroris/curation/actions/_mol.py

```diff
@@ -11,6 +12,11 @@
 from auroris.types import VerbosityLevel
 from auroris.visualization import visualize_chemspace
 
+try:
+    from molfeat.trans.pretrained.hf_transformers import PretrainedHFTransformer
+except Exception:
+    PretrainedHFTransformer = None
```

Comment:
General tip: Also good to be as precise as possible when catching exceptions. 

```suggestion
try:
    from molfeat.trans.pretrained.hf_transformers import PretrainedHFTransformer
except ImportError:
    PretrainedHFTransformer = None
```